src/main/kotlin/org/ozwillo/dcimporter/web/Routes.kt

```diff
@@ -33,6 +33,7 @@ class Routes(
                 POST("/{siret}/{request}", dataAccessRequestHandler::create)    //request = save for SAVED state, send for SENT state (meaning sent to validation task)
                 PUT("/{siret}/{action}/{id}", dataAccessRequestHandler::update)     //action = valid (push "Valider" button, update state to VALIDATED) or reject (push "Annuler" button, update state to REFUSED) or send (in case of pre-filled form sent to validation) or save (in case or updating pre-filled form)
                 GET("/{siret}/{state}", dataAccessRequestHandler::get)  //state = saved (list of saved pre-filled form) or sent (list of elements waiting for validation)
+                GET("/{id}", dataAccessRequestHandler::dataAccess)
```

Comment:
For your merge conflict : Here what do we do ? 
Do we keep the frontend-integration branch GET /{siret} -> In this case, only one function handle the getById and the getByState depending on the parameter entered in the url (?state=somethind -> getByState, ?id=something -> getById). In that case, the dataAccess() function in DataAccessRequestHandler.kt can be deleted

Or do we get two routes GET /{siret}/{state} and GET "/{id}" -> In this case, we keep the dataAccess() function but the get function in the frontend-integration branch must be modified to fit the GET /{siret}/{state} url

Reply:
Why we don't have a {siret} part in your GET /{id} url ?

Reply:
because the Siret isn't necessary there